### ISSUE TITLE
BUG Report: New command buttons are vbButtonGraphical by default

### ISSUE THREAD

**Issue by yereverluvinunclebert** [twinbasic/twinbasic#1955]
This is NOT really a bug, more of an observation that something is different from VB6, probably a good thing that it is.
New command buttons are vbButtonGraphical by default whereas in VB6 they are standard buttons (that don't like things like colour changes to background colours). I much prefer the graphical buttons and TBs default states for these. I raise this just in case it matters elsewhere.

**Comment by fafalone** [comment]
It's not like that for me... not even sure how that could happen.

Did you check if this happens in completely new projects, or it's just happening in one of your VB6 imports?

**Comment by WaynePhillipsEA** [comment]
I'm not sure how that would be possible.   The default properties should match VB6.

**Comment by yereverluvinunclebert** [comment]
I am running in VB6 and TB concurrently, imported the TB project a few days ago. When I added add a single new command button it defaulted to graphical. I created a control array of buttons from that original button. I didn't make the change to vbButtonGraphical myself. It is something I always forget to do but I didn't have to in this case, as it was set already. 

It was a colour change that I wanted to apply. I added the buttons in VB6 and they appeared as vbStandard buttons 
I wasn't able to copy/paste buttons (but I tried) so then I added the buttons in TB manually. Somehow it picked up vbGraphical...

I don't understand it either. Since I restarted TB's IDE new buttons are now vbStandard. Let's close this. Somehow, I must have accidentally switched the property value, I do have a cat and she does love keyboards.
